aikau/src/main/resources/alfresco/renderers/InlineEditProperty.js

```diff
@@ -476,6 +541,8 @@ define(["dojo/_base/declare",
        */
       onSave: function alfresco_renderers_InlineEditProperty__onSave(formPayload) {
          /*jshint unused:false*/
+         domClass.add(this.domNode, "alfresco-renderers-InlineEditProperty--updating");
```

Comment:
Just a casual aside, this one. Is there any point in using a setTimeout somewhere to be sure this displays?